src/screen/HtmlScreen.js

```diff
@@ -90,6 +90,19 @@ class HtmlScreen extends RequestScreen {
 		}
 	}
 
+	/**
+	 * Removes the favicon element from the document.
+	 * @param {!Array<Element>} resources 
+	 * @private
+	 * @return {CancellablePromise}
+	 */
+	clearFaviconInDoc_(resources) {
```

Comment:
@matuzalemsteles This method looks generic enough that it doesn't need to have the word "favicon" in it. This looks more like an utility method for removing an array of nodes. I would check if we already have a similar utility and, if not, add this to the senna utility folder and rename it to `removeElementsFromDocument` or something similar

Reply:
Yes, I think we can move this to `utils` and reuse elsewhere as well. Thanks for that.